### ISSUE TITLE
ASSE no desglosa datos de usuarios por departamento

### ISSUE THREAD

**Issue by fernandouval** [datauy/a-tu-servicio#1]

**Comment by picandocodigo** [comment]
Cierto, no tenemos ese dato, cambió con la reestructura de los datos de ASSE este año. Capaz que es algo que podemos conversar en la próxima reunión que tengamos con el MSP @danielcarranza @scrollif @Vitoartois

**Comment by danielcarranza** [comment]
Si, es para ver en el taller. ¿Pero no están tampoco en los datos crudos? Ojo que creo que sí.
ASSE tiene un problema de aquí está adaptando sistemas y midiendo desparejo en la medida que avanza el proyecto. recién en 2018 vamos a contar con datos completos y confiables.
